tests/modelling/test_us_equity_pricing_loader.py

```diff
@@ -97,7 +97,7 @@
 class BcolzDailyBarTestCase(TestCase):
 
     def setUp(self):
-        all_trading_days = TradingEnvironment.instance().trading_days
+        all_trading_days = TradingEnvironment().trading_days
```

Comment:
We probably shouldn't be creating new full `TradingEnvironments` in every test if we can avoid it.  I'd move this into a `setupClass` and share the environment between methods of this class.